.github/workflows/actions.yml

```diff
@@ -1,26 +1,21 @@
 name: Continuous Integration
-on:
-  pull_request:
-    branches:
-      - master
-  push:
-    branches:
-      - master
+on: push
+
 jobs:
   specs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         ruby:
-          - '3.1'
-          - '3.2'
           - '3.3'
+          - '3.4'
         gemfile:
-          - gemfiles/rails6.0.gemfile
-          - gemfiles/rails6.1.gemfile
-          - gemfiles/rails7.0.gemfile
           - gemfiles/rails7.1.gemfile
+          - gemfiles/rails7.2.gemfile
+          - gemfiles/rails8.0.gemfile
+        include:
+          - {ruby: '3.3', gemfile: 'gemfiles/rails7.0.gemfile'}
```

Comment:
maybe just drop rails 7.0 🤷

Reply:
Being able to bundle with Rails 7.0 useful (for a very brief moment), gonna release this then open a PR to drop 7.0.